src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -20,7 +20,7 @@ The HTTP instrumentation can lead to crashes due to a bug in the SDK (see [GitHu
 
 This feature is available for iOS, tvOS, and Mac Catalyst, and only works for UIViewControllers.
 
-The UIViewController Instrumentation, once enabled, captures transactions whenever your app loads a ViewController. The SDK sets the transaction name to the name of the ViewController, including the module — for example, `Your_App.MainViewController` — and the transaction operation to `ui.load`.
+The UIViewController Instrumentation, once enabled, captures transactions whenever your app loads an in-app UIViewController, meaning it doesn't capture transactions of UIViewControllers of UIKit, other 3rd party libraries, or SwiftUI. The SDK sets the transaction name to the name of the ViewController, including the module — for example, `Your_App.MainViewController` — and the transaction operation to `ui.load`.
```

Comment:
Should there be a comma between "of UIViewControllers" and "of UIKit"?

Reply:
Maybe `transactions of UIKits UIViewControllers` is better?

Reply:
Yes, I think that'll work better in the context of this list.